src/main/java/seedu/sudowudo/logic/commands/AddCommand.java

```diff
@@ -38,6 +38,8 @@
     private final Item toAdd;
     private Item toUndoAdd;
     private boolean hasTimeString = false;
+    
+    private final DateTimeParser dtparser = DateTimeParser.getInstance();
```

Comment:
Also organize variables:
1. Class (static) variables in the order public, protected, package (no access modifier), private
2. Instance variables in the order public, protected, package (no access modifier), private